listenbrainz/webserver/static/js/src/follow/SimilarUsersModal.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/prop-types */
```

Comment:
This isn't the way to go to fix an issue ;)

Here, removing the disable rule gives us an error that we should pay attention to: something not right with the props. The error itself is not very understandable, but it points to an issue with how we're using the props.

In this case, React doesn't like the pattern you're using: each time any of the props change, the `renderSimilarUsersList` function will be recreated. You can mamke a memoized version of it that will detect changes to certain props dependencies and only render if those dependencies have changed.

I will comment directly on the lines concerned